src/templates/finops-hub/schema/schema_ea_normalized.json

```diff
@@ -0,0 +1,723 @@
+{
+  "additionalColumns": [
+    {
+      "name": "costInPricingCurrency",
+      "value": "$$COLUMN:CostInBillingCurrency"
+    },
+    {
+      "name": "costInUsd",
+      "value": 0.0
```

Comment:
Should we set this to null instead of 0? 0 means there's no cost. Null means we don't have a value. Maybe it's semantics, but we don't want someone adding up the costInUsd column and getting a different value than costInBillingCurrency for a USD account.

Reply:
Done